### PR TITLE
[CHG] mozaik_mandate: allow to use wizard to close/reactivate several mandates together

### PR DESCRIPTION
**Pull request by marielejeune**
refs #65915

### CODE REVIEW

**Comment by marielejeune** [comment]
@RobinetDenisAcsone do you know why this wizard was only dealing with a single mandate?

File: mozaik_mandate/wizards/update_int_mandate_end_date_wizard.xml [hunk 1]
```diff
@@ -40,7 +42,7 @@
             name="groups_id"
             eval="[(6, 0, [ref('mozaik_abstract_model.res_groups_archiving_manager')])]"
         />
-        <field name="context">{'mode':'reactivate'}</field>
+        <field name="context">{'mode':'reactivate', 'active_test': False}</field>
```
Comment (marielejeune) [inline mozaik_mandate/wizards/update_int_mandate_end_date_wizard.xml]: If we do not pass this context key, `default_get` returns the ids of the mandates to reactivate but as they are archived, they are not taken into account in `mandate_ids` field of the wizard.
@RobinetDenisAcsone do you have a better solution?

Comment (RobinetDenisAcsone) [reply mozaik_mandate/wizards/update_int_mandate_end_date_wizard.xml]: Perhaps try something like `mandate_ids = fields.Many2many(comodel_name="ext.mandate", context={'active_test': False})`?
Not quite sure if this works.

**Comment by RobinetDenisAcsone** [comment]
@marielejeune I don't know it was already like that in v8

File: mozaik_mandate/wizards/update_int_mandate_end_date_wizard.xml (hunk 1 above)
Comment (marielejeune) [reply mozaik_mandate/wizards/update_int_mandate_end_date_wizard.xml]: Good idea, it works, thanks!

**Pull request merged**